src/app/controller/pipes/timestamp-to-standardized-timestamp.ts

```diff
@@ -37,23 +37,29 @@ export class TimestampToStandardizedtTimestampPipe implements PipeTransform {
     }
 
     newTimestamp.publisher = configurations.publisher;
-    newTimestamp.publisherRole = portcallTimestamp.timestampDefinition.publisherRole;
+    newTimestamp.publisherRole = portcallTimestamp.timestampDefinitionTO.publisherRole;
     newTimestamp.vesselIMONumber = portcallTimestamp.vesselIMONumber;
+    newTimestamp.vessel = portcallTimestamp.vessel;
+    newTimestamp.carrierServiceCode = portcallTimestamp.carrierServiceCode;
+    newTimestamp.exportVoyageNumber = portcallTimestamp.exportVoyageNumber;
+    newTimestamp.importVoyageNumber = portcallTimestamp.importVoyageNumber;
+    newTimestamp.portVisitReference = portcallTimestamp.portVisitReference;
+    newTimestamp.carrierExportVoyageNumber = portcallTimestamp.carrierExportVoyageNumber;
+    newTimestamp.carrierExportVoyageNumber = portcallTimestamp.carrierExportVoyageNumber;
```

Comment:
duplicate line

Reply:
assume it should be carrierImportVoyageNumber

Reply:
This is a mistake

Reply:
Yeah, for our case it only makes difference when we defining a new Type for example. Only reason I add them is for reference & clarification. It has no real affect.